README.md

```diff
@@ -142,17 +143,18 @@ This step will install the following services:
 
 
 7. Prepare the environment
-1) Create a namespace for cluster management, it should be 'cluster-system'.
+
+    7.1 Create a namespace for cluster management, it should be 'cluster-system'.
 
     ```
     kubectl create -n cluster-system
     ```
 
-2) Add current cluster to the portal. Navigate to '集群管理' and '添加集群'
-* for API Host, use the one from `hostK8sApiWithOidc`
-* for API Token, use the one you saved from step 3.
+    7.2 Add current cluster to the portal. Navigate to '集群管理' and '添加集群'
+    * for API Host, use the one from `hostK8sApiWithOidc`
+    * for API Token, use the one you saved from step 3.
```

Comment:
here it metioned api token which can be found from step3.Somehow which step is `step3`?